R/rdeephaven/R/table_handle_wrapper.R

```diff
@@ -1,43 +1,91 @@
-# TODO: Document this guy
-
+#' @title Deephaven TableHandles
+#' @description Deephaven TableHandles are references to tables living on a Deephaven server. They provide an
+#' interface for interacting with tables on the server.
+#' 
+#' @usage NULL
+#' @format NULL
+#' @docType class
+#' 
+#' @section Methods
+#' 
+#' - `$bind_to_variable(name)`
+#' - `$to_arrow_record_batch_stream_reader()`
+#' - `$to_arrow_table()`
+#' - `$to_tibble()`
+#' - `$to_data_frame()`
+#'
+#' @examples
+#' 
+#' # connect to the Deephaven server running on "localhost:10000" using anonymous 'default' authentication
+#' client_options <- ClientOptions$new()
+#' client <- Client$new(target="localhost:10000", client_options=client_options)
+#' 
+#' # open a table that already exists on the server
+#' new_table_handle1 <- client$open_table("table_on_the_server")
+#' 
+#' # convert the Deephaven table to an R data frame
```

Comment:
data frame vs. dataframe. Which is preferred? Let's stay consistent

Reply:
I believe I've consistently chosen "data frame", cannot find instances where that is not the case